docker-compose.yml

```diff
@@ -2,7 +2,8 @@ version: "3.4"
 
 services:
   listingapi:
-    build: listingapi
+    image: ghcr.io/meilleursagents/backend-technical-test/listingapi:latest
+    user: "listingapi:group_listingapi"
```

Comment:
Most of the time it is conventional to have the group matching the user, do you see situations where having the `group_` prefix can be useful?

Reply:
Nope.